src/bin/sail.ml

```diff
@@ -526,6 +527,15 @@ let parse_config_file file =
     Reporting.warn "" Parse_ast.Unknown (Printf.sprintf "Failed to parse configuration file: %s" message);
     None
 
+exception VersionParseError of string
```

Comment:
It might be better to just raise a general error with `raise (Reporting.err_general ...)`, rather than introduce a special exception type.